lib/unicore/mktables

```diff
@@ -17556,7 +17557,7 @@ END
             my $L = $LBase + $SIndex / $NCount;
             my $V = $VBase + ($SIndex % $NCount) / $TCount;
             my $T = $TBase + $SIndex % $TCount;
-            $name = "$HANGUL_SYLLABLE$Jamo{$L}$Jamo{$V}";
+            my $name = "$HANGUL_SYLLABLE$Jamo{$L}$Jamo{$V}";
```

Comment:
atoomic already tried this, and it didn't work IIRC

Reply:
@atoomic and I made this change together. I'm not aware of a problem.